website/docs/for-capacitor/module-federation.md

```diff
@@ -0,0 +1,82 @@
+---
+title: How to use with Module Federation
+sidebar_label: How to use with Module Federation
+---
+
+The first thing you will need to do when implementing Portals for Capacitor is choose a web based micro frontend solution. The most common is [Module Federation](https://webpack.js.org/concepts/module-federation/).
+
+After you have chosen and setup module federation between your apps you will need to configure it for how to build for Capacitor. The easiest way to do this is to add an environment variable to your npm build script.
+
+In this case we are setting the variable `CAP_BUILD` to true. In the example below we are using `cross-env` to set the environment variable within our npm build script.
+
+```javascript
+"scripts: {
+  "build": "cross-env CAP_BUILD=true nx run-many --target=build --all",
+  ...
+}
+```
+
+This variable is then accessed within your module federation configuration as a way to identify when to use the Capacitor paths for your remotes. You can see below that we have three different MFEs that we are pulling into our primary Capacitor application (`account`, `checkout`, `helpinfo`).
+
+```javascript
+const isCapBuild = process.env.CAP_BUILD;
+
+const remotes =
+  isCapBuild == null
+    // Configuration for when developing locally
+    ? {
+        account: 'account@http://localhost:3004/remoteEntry.js',
+        checkout: 'checkout@http://localhost:3005/remoteEntry.js',
+        helpinfo: 'helpinfo@http://localhost:3006/remoteEntry.js',
+      }
+    // Configuration for when building for capacitor
+    : {
+        account: `account@account/remoteEntry.js`,
+        checkout: `checkout@checkout/remoteEntry.js`,
+        helpinfo: `helpinfo@helpinfo/remoteEntry.js`,
+      };
+
+module.exports = {
+  name: 'shell',
+  remotes,
+  ...
+}
+```
+
+## Capacitor Configuration
+
+The next step is to configure capacitor so that it knows where these files exist on the initial application build. You can see that there is configuration for the shell or main application and then config for the MFEs that are separate. the webDir paths are relative your the base directory of the shell application (where this configuration file lives). The names correspond to how you named the directories within the module federation config (`account`, `checkout`, `helpinfo`).
```

Comment:
```suggestion
The next step is to configure Capacitor so that it knows where these files exist on the initial application build. You can see that there is configuration for the shell or main application and then config for the MFEs that are separate. the webDir paths are relative your the base directory of the shell application (where this configuration file lives). The names correspond to how you named the directories within the module federation config (`account`, `checkout`, `helpinfo`).
```